### PR TITLE
IE11 fixes

### LINKED ISSUE #99: Home view expands past 100% window width
Problem: Home view expands past 100% window width showing horizontal scrollbar

Environment: Chrome and Windows

image

### LINKED ISSUE #124: Website doesn't work using Internet Explorer 11
Page doesn't load, details in screenshot.

image

### PR DESCRIPTION
**Pull request by nylira**
- [x] Website is now confirmed to load on IE11
- [x] Website homepage no longer has a horizontal scrollbar when scrollbar is enabled
- [x] Resolve `window.atob` `Unhandled promise rejection InvalidCharacterError` for IE11
- [x] Fix roadmap display for IE11

## Old

virtualbox_win8 ie11_23_02_2018_15_34_31

## New

virtualbox_win8 ie11_23_02_2018_15_52_50

virtualbox_win8 ie11_23_02_2018_14_56_21

Closes #124 
Closes #99

### CODE REVIEW

**Comment by nylira** [comment]
It's ridiculous to try and fix flexbox bugs for IE11, when IE devs themselves marked the flexbox bugs as `wontfix`. So anyone using IE11 will get a degraded experience with the roadmap, but all of the correct data is there.

**Comment by faboweb** [comment]
Thanks Peng for going the extra mile! we should probably think about cross browser testing.

**Comment by nylira** [comment]
Yes I'm attempting to sign up Cosmos for BrowserStack

File: index.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
 <title>Internet of Blockchains - Cosmos Network</title>
 <meta name=viewport content="width=device-width, user-scalable=no">
 <!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
+<!--[if lt IE 12]><script src="https://cdn.jsdelivr.net/npm/promise-polyfill@7/dist/polyfill.min.js"></script><![endif]-->
+<script src="https://cdn.jsdelivr.net/npm/promise-polyfill@7/dist/polyfill.min.js"></script>
```
Comment (haasted) [inline index.html]: Isn't the polyfill being loaded under all circumstances like this?

**Pull request merged**